.github/workflows/build-ale.yml

```diff
@@ -53,8 +53,11 @@ jobs:
       if: matrix.os != 'windows-latest'
       run: |
         mkdir build; cd build
-        cmake ../ -DUSE_SDL=ON
+        cmake ../ -DUSE_SDL=ON -DPython3_FIND_FRAMEWORK=NEVER
     - name: Build & Test ale
       run: |
         cd build
-        cmake --build . --config Release --target tests
+        cmake --build . --config Debug
+        ctest -C Debug --progress -VV
```

Comment:
There's a known bug with VSCode that prevents us from using `cmake --build . --target test` so we'll use `ctest` for the time being.